### PR TITLE
Provide an overview into digital advertising

### PR DESCRIPTION
**Pull request by nphase**
As promised, I have an early draft of a “marketing 101”-level overview, which attempts to describe common forms of digital advertising, and the component pieces therein.

I realize this is too late to be discussed in today’s meeting, but I hope it serves as a useful primer and set-up device to explain what business purposes some of the technology decisions we’ve made serve. 
More to come on this end.

I welcome the contributions of others in helping to clarify some of the commercial context.

### CODE REVIEW

File: advertising101.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  * When a campaign is multi-channel (which almost all of them are), the brand survey provider will accommodate for the different combinations of exposures across the various different channels and providers, which means that they will need all partners i.e. the DSP, the social providers, and the OLV providers to provide their exposed users using their respective identifiers and identity graphs
+
+
+*Consumer*: I am looking for diapers that fit, so to see ads that solve my specific problem is not intrusive (assuming the ad format isn’t annoying). If these ads follow me around for a relatively short period of time, I’m okay with that. If I had to instead pay for the publisher.com content and not see any ads, that would be okay as long as that was my choice. The same with my relationship with browser.com  goal is to show
```
Comment (scottlow) [inline advertising101.md:107]: > The same with my relationship with browser.com  goal is to show

This seems to trail off. Not sure if there was more to this sentence?

File: advertising101.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+* I repeat the steps from relationship marketing, but this time elect to trial different messages using both dynamic creative optimization and adaptive segments because I want to react in real time based on his behaviors, tailoring my campaign and message based on what he interacts with or not
+* I also set higher frequency caps and reach out to “him” on more digital channels than I have previously (via the segment – everyone in the segment is treated the same initiall,y and again, I don’t understand tindividuals tied to my CRM profiles). I can do this because my DSP has good understanding of the various devices attached to the individuals in my segment
+* Based on attribution run through the DSP using the household level identity graph, the DSP is able to run lift measurement and understand that the ads led to a conversion at the household level, not at the individual level (the individual who saw the ads is not the one who made the purchase, but the HH did make a purchase)
```
Comment (scottlow) [inline advertising101.md:146]: What data signals do DSPs use to identify households? I'm guessing location/IP, but are there others?

File: advertising101.md (hunk 1 above)
Comment (nphase) [reply advertising101.md:107]: Ooops! corrected.

File: advertising101.md (hunk 2 above)
Comment (nphase) [reply advertising101.md:146]: Will make sure to cover this in subsequent iterations which ladder down to technical mechanisms that implement business goals!

**Pull request merged**